docs/topics/making-open-source-more-inclusive.adoc

```diff
@@ -10,6 +10,7 @@
 // * docs/web-console-guide/master.adoc
 
 [preface]
+:_content-type: SNIPPET
 [id="making-open-source-more-inclusive_{context}"]
```

Comment:
This is a concept module. As far as I know, snippets don't have IDs, headings, or role assignments.

Reply:
I was not sure about this. It does have an ID and a heading. But on the other hand, it's sort of boilerplate used in several places.

Reply:
@sbeskin-redhat This is a concept. Please change it.